resources/slack-openapi-sorted.patch

```diff
@@ -26,6 +26,7 @@
                  "created",
                  "id",
                  "is_intro",
+\ No newline at end of file
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏I think this should not appear in the patch file :thinking: